docs/neo-n3/dapp_development/preliminaries.md

```diff
@@ -1,35 +1,15 @@
 # Preliminaries
 
-The following sections introduce general concepts and types that you will meet throughout the neow3j SDK.
-## Hashes
+The following sections introduce general concepts and types that you will encounter within the neow3j SDK.
 
-Hashes appear a lot in the blockchain world and Neo is not an exception. There are two kinds of hashes in Neo. One
-is 256 bit long and produced by applying the SHA-256 (member of the SHA-2 cryptographic hash functions) twice to some
-input data. It is used for transaction and block hashes. The other is 160 bit long and produced by first applying
-SHA-256 and then RIPEMD-160 to some input data. It is used for identifying accounts and contracts in the form of script
-hashes, i.e., hashes of the scripts underlying those accounts and contracts.  
+## Hashes
 
-The neow3j SDK uses the types `io.neow3j.contract.Hash256` and `io.neow3j.contract.Hash160` for those two hash types,
-respectively. You will see that the SDK's API almost always requires you to work with these two types instead of a
-simple string or byte array. Note, that the hashes stored by these types are stored in big-endian order but can be
-retrieved in little-endian order via the `toLittleEndianArray()` method. Endianness might be an issue when inspecting
-results from contract invocations, because the Neo node might return a hash in little-endian order, which you have to be
-aware of when constructing a `Hash160` or `Hash256`.
+Hashes are frequently used in the blockchain world, and Neo is no exception. There are two types of hashes in Neo. One is 256 bits long and is generated by applying the SHA-256 algorithm (a member of the SHA-2 cryptographic hash functions) twice to certain input data. This type of hash is used for transaction and block hashes. The other type is 160 bits long and is generated by first applying SHA-256 and then RIPEMD-160 to specific input data. This hash type is used for identifying accounts and contracts in the form of script hashes, which are hashes of the scripts associated with those accounts and contracts.
 
+The neow3j SDK uses the types `io.neow3j.contract.Hash256` and `io.neow3j.contract.Hash160` for these two types of hashes, respectively. You will find that the SDK's API typically requires you to work with these specific types rather than a simple string or byte array. It's important to note that the hashes stored by these types are stored in big-endian order, but they can be retrieved in little-endian order using the `toLittleEndianArray()` method. Endianness can be a concern when inspecting results from contract invocations because the Neo node might return a hash in little-endian order. This difference in endianness must be taken into account when constructing a `Hash160` or `Hash256` object.
```

Comment:
@thedanielmark I think it'd be nice to link `Endianness` to https://en.wikipedia.org/wiki/Endianness

Then, maybe some developer can refresh their memory.